### PR TITLE
Change directories structure

### PR DESCRIPTION
**Pull request by vcerenu**
This PR modifies the directory structure of the repository, to order the deployment of multi-node and single-node stacks, in addition to organizing the build of Wazuh Docker images.

### CODE REVIEW

File: multi-node/volume-migrator.sh [hunk 1]
```diff
@@ -0,0 +1,279 @@
...
+           alpine ash -c "cd /from ; cp -avp . /to"
+
+docker container run --rm -it \
+           -v wazuh-docker_worker-filebeat-etc:/from \
+           -v $2_worker-filebeat-etc:/to \
+           alpine ash -c "cd /from ; cp -avp . /to"
+
+docker container run --rm -it \
+           -v wazuh-docker_worker-filebeat-var:/from \
+           -v $2_worker-filebeat-var:/to \
```
Comment (teddytpc1) [inline multi-node/volume-migrator.sh:278]: Nice!

File: README.md [hunk 2]
```diff
@@ -87,73 +87,95 @@ ADMIN_PRIVILEGES=true               # App privileges
...
+│   │   │   └── wazuh_worker.conf
+│   │   ├── wazuh_dashboard
+│   │   │   ├── opensearch_dashboards.yml
+│   │   │   └── wazuh.yml
+│   │   ├── wazuh-indexer
+│   │   │   ├── internal_users.yml
+│   │   │   ├── wazuh1.indexer.yml
+│   │   │   ├── wazuh2.indexer.yml
+│   │   │   └── wazuh3.indexer.yml
+│   │   └── wazuh_indexer_ssl_certs
```
Comment (teddytpc1) [inline README.md:155]: Update tree (without cert files)

**Review by teddytpc1 — CHANGES_REQUESTED** [review]

**Review by teddytpc1 — APPROVED** [review]
LGTM

**Review by alberpilot — CHANGES_REQUESTED** [review]
- The folder `indexer_certs_creator` doesn't match the other folder format name. `indexer-certs-creator`.
- Remove empty lines from the main README variables sections. 
- The directory structure is broken
- Add README under `multi-node`, `single-node`, `build-docker-images`. The new `indexer-certs-creator` should have a README too. 
- Additionally `generate-indexer-certs.yml` doesn't work:
```
~/repositories/wazuh-docker/multi-node   change-directories ●  docker-compose -f generate-indexer-certs.yml up                
Creating network "multi-node_default" with the default driver
Pulling generator (wazuh/wazuh-certs-generator:0.0.1)...
0.0.1: Pulling from wazuh/wazuh-certs-generator
08c01a0ec47e: Pull complete
6d34cc3bb0a2: Pull complete
bcad6faac775: Pull complete
c7fbc190910a: Pull complete
b747db0e8163: Pull complete
37cacab0e3bb: Pull complete
Creating multi-node_generator_1_a2e47470a1d4 ... done
Attaching to multi-node_generator_1_c9056ee06162
generator_1_c9056ee06162 | 31/03/2022 09:15:19 ERROR: No configuration file found. /unattended_installer/install_functions/config.yml.
generator_1_c9056ee06162 | Moving created certificates to destination directory
generator_1_c9056ee06162 | cp: cannot stat '/unattended_installer/install_functions/certs/*': No such file or directory
generator_1_c9056ee06162 | changing certificate permissions
```

**Review by teddytpc1 — APPROVED** [review]
LGTM

**Review by alberpilot — APPROVED** [review]

**Pull request merged**